pkg/security/resolvers/tags/resolver.go

```diff
@@ -98,3 +100,41 @@ func NewResolver(config *config.Config) *Resolver {
 		tagger: &nullTagger{},
 	}
 }
+
+// Resolove image_id
+func (t *Resolver) ImageIDResolver(containerID string) string {
+	m := workloadmeta.GetGlobalStore()
+
+	// Get imageID from container so that we can get the image Metadata
+	container, err := m.GetContainer(containerID)
+	if err != nil {
+		log.Errorf("unable to get container %s: %s", containerID, err)
```

Comment:
We need to return here.
You can change your function to make it return both a string and a error
and display the log in the calling function